### DISCUSSION TITLE
Possible error in TCPairs use case

### DISCUSSION THREAD

**Discussion by j-opatz** [dtcenter/METplus#2625] [Use Cases] [answered]
@mkavulich noted in a separate Discussions post: I did notice a minor error in the basic TCPairs use case: the provided conf file sets TC_PAIRS_DLAND_FILE, but this was not the correct setting; I had to instead set METPLUS_DLAND_FILE to get the run to complete. Should I open a separate issue for this correction?

**Comment by j-opatz** [comment]
Hi @mkavulich,

Given the subject area of this Discussions topic, I'm going to pull in @mrinalbiswas as our SME. I know they'll be able to sort out why you're experiencing an issue with TC_PAIRS_DLAND_FILE and not METPLUS_DLAND_FILE.

**Comment by georgemccabe** [comment, marked as answer]
Hi @mkavulich,

The METplus config variable is [TC_PAIRS_DLAND_FILE](https://metplus.readthedocs.io/en/latest/Users_Guide/glossary.html#term-TC_PAIRS_DLAND_FILE), so the TCPairs basic use case config file is correct. The environment variable that is set by the METplus wrappers before calling the tc_pairs tool is named `METPLUS_DLAND_FILE`, but setting this variable in either your METplus config file or the terminal will not have any effect on the value used by METplus wrappers.

If changing `TC_PAIRS_DLAND_FILE` to `METPLUS_DLAND_FILE` in the METplus config file fixed the failure, then it sounds like there may be an issue finding the dland file. Your change may cause the use case to succeed, but it may not be using the dland file as intended.

Could you please rerun METplus before changing `TC_PAIRS_DLAND_FILE` to `METPLUS_DLAND_FILE` and with the addition of `LOG_LEVEL=DEBUG`, then provide me with the log file from your run?

**Reply by mkavulich** [reply]
@georgemccabe Thanks for investigating and for your response. I have played around with my setup a bit more, and I find myself unable to replicate the problem I originally reported. So your theory may be correct, that I had simply been pointing to an incorrect file and misinterpretws the error message. Regardless, I think it is okay to close this discussion as "resolved". Thanks again!

**Reply by georgemccabe** [reply]
Thanks for the info, @mkavulich. I'm going to lock this discussion to prevent future posts. If you encounter this issue again, please create a new discussion and tag me so I can investigate further. Thanks!
